client/src/components/propertyhome/PropertyHome.js

```diff
@@ -38,10 +38,9 @@ function PropertyHome() {
           <PropertyDetailCard address={property.address} id={property._id} name={property.name} />
           <div className="property-actions-container">
             <div className="property-actions">
-              <button className="property-action">Edit Tenants</button>
+              <button className="property-action">Add Tenants</button>
```

Comment:
maybe change this to "Add a tenant"? 

since users can add **at most 1 tenant** everytime